index.bs

```diff
@@ -407,4 +434,30 @@ This callback will be invoked when there are changes to <i>target</i>'s intersec
                 </ol>
             </ul>
         </ul>
+
+<h3 id='3-3-external-spec-integrations'>External Spec Integrations</h3>
+
+<h4 id='3-3-1-event-loop'>HTML Processing Model: Event Loop</h4>
+    An <a>Intersection Observer</a> processing step should take place AFTER the
+    layout and rendering steps have been performed in the
+    <a href="https://html.spec.whatwg.org/multipage/webappapis.html#processing-model-8">
```

Comment:
Right, that makes sense, and I agree. All I'm saying is that given that you're linking to https://html.spec.whatwg.org/multipage/ here, you should change the link defaults (at the top of the .bs file) from http://www.w3.org/TR/html5/ to https://html.spec.whatwg.org/multipage/.